packages/fluentui/docs/src/index.tsx

```diff
@@ -1,14 +1,9 @@
+import 'react-hot-loader';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import { AppContainer } from 'react-hot-loader';
 import App from './app';
 
 const mountNode = document.createElement('div');
 document.body.appendChild(mountNode);
 
-ReactDOM.render(
-  <AppContainer>
```

Comment:
`AppContainer` is redundant, it's enough to use [`hot()`](https://github.com/gaearon/react-hot-loader#getting-started) and we are already doing that.